cpp/util/libevent_wrapper.h

```diff
@@ -20,6 +20,11 @@
 #include "util/executor.h"
 #include "util/task.h"
 
+#define MAX_HTTP_HEADER_SIZE (ev_ssize_t)4096
+#define MAX_HTTP_POST_BODY_SIZE (ev_ssize_t)32768
```

Comment:
Should move those to `libevent_wrapper.cc`, inside the anonymous namespace, since they are not needed anywhere else. Also, you should make them C++ variables rather than C preprocessor macros.

Take note of there relevant part of the style guide:

https://google.github.io/styleguide/cppguide.html#Constant_Names

So it could be something like `const ev_ssize_t kMaxHttpHeaderSize = 4096;`, say?